features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/metric/slowframes/DefaultSlowFramesListenerTest.kt

```diff
@@ -414,6 +430,13 @@ internal class DefaultSlowFramesListenerTest {
     )
 
     companion object {
+
+        fun DefaultSlowFramesListener.assertResolveNonNullReport(viewId: String): ViewUIPerformanceReport {
+            val report = resolveReport(viewId)
+            assertThat(report).isNotNull
+            return report!!
+        }
```

Comment:
minor: is this function needed? normally it can be just `val report = checkNotNull(resolveReport(viewId))`